lib/tasks/coverage.rake

```diff
@@ -14,7 +14,7 @@ namespace :generate do
       funs.each do |fun|
         counter += 1
         text = text.sub(fun, "*/module_#{counter}") # Replaces the function with module_{number}
-        fun = fun.gsub(%r/\/[*]{3}\/ \(function/, "var module_#{counter} = (function")
+        fun = fun.gsub(%r/\/[*]{3}\/ \(/, "var module_#{counter} = (")
```

Comment:
@ragesoss Need to validate if `coverage.rake` file is working as expected since the output in `public/assets/javascripts/main.js` might have changed from webpack 4 to 5.

Reply:
Yep, I tried it out locally and the coverage.rake is still working.